### ISSUE TITLE
Accessibility labels missing from language selection screen

### ISSUE THREAD

**Issue by jleach** [openwallet-foundation/bifold-wallet#824]
### Description of problem

The radio buttons in the language selection screen do not have accessibility labels.

### Expected behavior

Each label should have the accessibility label corresponding ot the language.

### Steps to reproduce

1. Go to Settings
2. Select the current language as if you want to change it;
3. On iOS say "Show Names" to VoiceControl. 
4. No labels will be displayed for the radio buttons.

### Screenshots and/or log output

![Screenshot 2023-06-14 at 8.56.36 AM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/4af774f0-edfb-4759-b9eb-9b05e40b4c36)

### Environment
- [ ] Occurs on Android
- [ ] Occurs on iOS

Build #:

Android Device Model: 

iOS Device Model: 

iPhone 14 Pro

### Severity
- [ ] High
- [ ] Medium
- [X] Low
